Zend/zend_interfaces.stub.php

```diff
@@ -6,44 +6,44 @@ interface Traversable {}
 
 interface IteratorAggregate extends Traversable
 {
-    /** @return Traversable */
-    public function getIterator();
+    /** @tentative-return-type */
+    public function getIterator(): Traversable;
 }
 
 interface Iterator extends Traversable
 {
     /** @return mixed */
-    public function current();
+    public function current(); // TODO Can't add tentative return tyoe due to DirectoryIterator::current()
 
-    /** @return void */
-    public function next();
+    /** @tentative-return-type */
+    public function next(): void;
 
     /** @return mixed */
-    public function key();
+    public function key(); // TODO Can't add tentative return tyoe due to DirectoryIterator::key()
 
-    /** @return bool */
-    public function valid();
+    /** @tentative-return-type */
+    public function valid(): bool;
 
-    /** @return void */
-    public function rewind();
+    /** @tentative-return-type */
+    public function rewind(): void;
 }
 
 interface ArrayAccess
 {
-    /** @return bool */
-    public function offsetExists(mixed $offset);
+    /** @tentative-return-type */
+    public function offsetExists(mixed $offset): bool;
 
     /**
      * Actually this should be return by ref but atm cannot be.
-     * @return mixed
+     * @tentative-return-type
      */
-    public function offsetGet(mixed $offset);
+    public function offsetGet(mixed $offset): mixed;
 
-    /** @return void */
-    public function offsetSet(mixed $offset, mixed $value);
+    /** @tentative-return-type */
+    public function offsetSet(mixed $offset, mixed $value): void;
 
     /** @return void */
-    public function offsetUnset(mixed $offset);
+    public function offsetUnset(mixed $offset); // TODO Can't add tentative return type due to Phar::offsetUnset
```

Comment:
I've adjusted this in https://github.com/php/php-src/commit/edb6b375dd8ecfd9711780068086524a59853ab6.